### PR TITLE
feature6 added - statistic

### PR DESCRIPTION
**Pull request by krzysiekpom**
The admin panel should display a number which shows the amount of interests in the system of "health" type, chosen by women aged between 20 and 30, and the names of interests should start with "cosm". The query should be written using SQL or appriopriate DSL offered for instance by ActiveRecord. Usng Ruby for data manipualtion here is a faux pas. In other words, show me your SQL magic. Requirement: you should write a query with a subquery for this task.

### CODE REVIEW

File: app/helpers/admin_helper.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+module AdminHelper
+  def statistic
```
Comment (socjopata) [inline app/helpers/admin_helper.rb]: Consider using Query Object pattern here. Be sure to include some specs for a query object you write.

**Comment by krzysiekpom** [comment]
I have wrapped the code in a Query Object and deleted the admin helper - I hope this is what you had in mind.

File: app/views/admin/index.html.erb [hunk 2]
```diff
@@ -2,7 +2,7 @@
 
 <div class="admin_header">
   <h3 class="admin">Admin Panel</h3>
-  <h3 class="statistic"><%= statistic %></h3>
+  <h3 class="statistic"><%= StatisticQuery.new.count %></h3>
```
Comment (socjopata) [inline app/views/admin/index.html.erb]: Very nice. As a bonus, consider rewriting the StatisticQuery, so you can call it like `StatisticQuery.count`, but underneath, you're still working on an instance of a class.

**Comment by krzysiekpom** [comment]
I must say I had a hard time doing this. I created the solution that would work without `#new` and pushed it, but then I realized this is not what you asked for - I made `#count` a class method. Then I tried to introduce the same but with instance method and the solution I came up feels a bit dirty to me.  As PR was already merged at this point it's not visible in the commits here, the commit with instance method changes is 365c829 (head on feature6).

**Pull request merged**